src/html5_playback.js

```diff
@@ -82,7 +86,9 @@ export default class HTML5TVsPlayback extends Playback {
     const currentSource = this.$sourceElement && this.$sourceElement.src
     if (sourceURL === currentSource) return
 
-    this._setSourceOnVideoTag(sourceURL)
+    this.config && this.config.drm && this.config.drm.licenseServerURL && !this._drmConfigured
+      ? DRMHandler.sendLicenseRequest.call(this, this.config.drm, this._onDrmConfigured)
```

Comment:
What if `sendLicenseRequest` fails?
Should we trigger a `PLAYBACK_ERROR`?
I think we need to create an error handler for this scenario

Reply:
You're right! I've applied your suggestions on those commits: da172f5fc7dc13a465011cccda7b01adcc81b3aa 91d51f816f6202fb3a3b81f3cc049ff703cc2444